compiler/src/main/java/bullet/impl/ComponentProcessingStep.java

```diff
@@ -224,4 +272,22 @@ private boolean isVisibleFrom(Element target, PackageElement from) {
         throw new AssertionError();
     }
   }
+
+  private static int getNextPrime(int value) {
```

Comment:
BigInteger.nextProbablePrime will do this for you and should be reliable for the small integers we are talking about here.